src/main/java/io/reactivex/parallel/ParallelFlowable.java

```diff
@@ -0,0 +1,701 @@
+     * @return the new ParallelFlowable instance
+     */
+    @CheckReturnValue
+    public final ParallelFlowable<T> runOn(Scheduler scheduler, int prefetch) {
+        ObjectHelper.requireNonNull(scheduler, "scheduler");
+        ObjectHelper.verifyPositive(prefetch, "prefetch");
+        return new ParallelRunOn<T>(this, scheduler, prefetch);
+    }
+
+    /**
+     * Reduces all values within a 'rail' and across 'rails' with a reducer function into a single
+     * sequential value.
+     * <p>
+     * Note that the same reducer function may be called from multiple threads concurrently.
+     * @param reducer the function to reduce two values into one.
+     * @return the new Px instance emitting the reduced value or empty if the ParallelFlowable was empty
+     */
+    @CheckReturnValue
+    public final Flowable<T> reduce(BiFunction<T, T, T> reducer) {
+        ObjectHelper.requireNonNull(reducer, "reducer");
+        return RxJavaPlugins.onAssembly(new ParallelReduceFull<T>(this, reducer));
+    }
+
+    /**
+     * Reduces all values within a 'rail' to a single value (with a possibly different type) via
+     * a reducer function that is initialized on each rail from an initialSupplier value.
+     * <p>
+     * Note that the same mapper function may be called from multiple threads concurrently.
+     * @param <R> the reduced output type
+     * @param initialSupplier the supplier for the initial value
+     * @param reducer the function to reduce a previous output of reduce (or the initial value supplied)
+     * with a current source value.
+     * @return the new ParallelFlowable instance
+     */
+    @CheckReturnValue
+    public final <R> ParallelFlowable<R> reduce(Callable<R> initialSupplier, BiFunction<R, T, R> reducer) {
+        ObjectHelper.requireNonNull(initialSupplier, "initialSupplier");
+        ObjectHelper.requireNonNull(reducer, "reducer");
+        return new ParallelReduce<T, R>(this, initialSupplier, reducer);
+    }
+
+    /**
+     * Merges the values from each 'rail' in a round-robin or same-order fashion and
+     * exposes it as a regular Publisher sequence, running with a default prefetch value
+     * for the rails.
+     * <p>
+     * This operator uses the default prefetch size returned by {@code Flowable.bufferSize()}.
+     * <img width="640" height="310" src="https://raw.github.com/wiki/ReactiveX/RxJava/images/rx-operators/parallelflowable.sequential.png" alt="">
+     * <dl>
+     *  <dt><b>Backpressure:</b></dt>
+     *  <dd>The operator honors backpressure.</dd>
+     *  <dt><b>Scheduler:</b></dt>
+     *  <dd>{@code sequential} does not operate by default on a particular {@link Scheduler}.</dd>
+     * </dl>
+     * @return the new Px instance
+     * @see ParallelFlowable#sequential(int)
+     */
+    @BackpressureSupport(BackpressureKind.FULL)
+    @SchedulerSupport(SchedulerSupport.NONE)
+    @CheckReturnValue
+    public final Flowable<T> sequential() {
```

Comment:
I think this should be a verb: `sequentize()`/etc to be consistent with other operators (which are verbs mostly).

Btw, reading chains like:

```java
Flowable.range(1, 100)
.parallel(10)
.runOn(Schedulers.io())
.map(v -> httpClient.blockingGet("http://server/item/" + v))
.sequential()
```

feels strange because `sequential` after `parallel`  looks like an operator that **disables** parallelization of the chain (of course it can't, but I dunno, it just reads strange to me).